src/main/scala/org/zalando/nakadi/client/KlientBuilder.scala

```diff
@@ -12,9 +12,14 @@ object KlientBuilder{
       new KlientBuilder(endpoint, port, securedConnection, tokenProvider, objectMapper)
 }
 
-class KlientBuilder(val endpoint: URI = null, val port: Int, val securedConnection: Boolean, val tokenProvider: () => String = null, val objectMapper: ObjectMapper = null)
+class KlientBuilder private (val endpoint: URI = null, val port: Int, val securedConnection: Boolean, val tokenProvider: () => String = null, val objectMapper: ObjectMapper = null)
```

Comment:
Placing `private` just there affects the constructor scope. This is useful since the construction is only intended to be done via actor initiation.